src/NextGen/CustomFields/Controllers/DonorDetailsController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Give\NextGen\CustomFields\Controllers;
+
+use Give\Donations\Models\Donation;
+use Give\Donors\Models\Donor;
+use Give\NextGen\CustomFields\Views\DonorDetailsView;
+use Give\NextGen\DonationForm\Models\DonationForm;
+use Give_Donor as LegacyDonor;
+
+/**
+ * @unreleased
+ */
+class DonorDetailsController
```

Comment:
I noticed these details classes are being called "controllers" but they just have a single invokable method - which I think lends itself to be classified as an action.  Do you foresee adding crud operations to these in the future? If that was the case I could see these `_invoke` methods being called `show`. Otherwise I think some more descriptive actions would work better like `ShowCustomFieldsInDonorDetails`

Reply:
These are kinda like single method invokable controller action things. So I grabbed a couple of those words by random 😅 For real though, I went back and fourth a lot and went with `Controller`. I like your idea of changing `__invoke` to `show` to be more controller like - I think that makes sense here.

Reply:
lol I can relate, naming things is hard!

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/99/commits/ba0ab4964789ad831fb31568a9e6f99e73b6e18e.